lib/ui/dashboard.js

```diff
@@ -514,11 +514,53 @@ class Dashboard {
       return { promise, cancellation: request.cancellation };
     }
 
+    const validElements = {
+      pods: "Pods",
+      logs: "Logs"
+    }
+
     this.render = function () {
       screen.append(pods_table);
       screen.append(resources);
       screen.append(pod_log);
-      pods_table.focus();
+
+      screen.key(['o', 'p'], function (_, key) {
```

Comment:
I wonder if we could just use <kbd>TAB</kbd> and <kbd>⇧</kbd>+<kbd>TAB</kbd> to be more generic?

Reply:
It may be necessary to remove the handler when the dashboard view is not active.